types/electron-packager/index.d.ts

```diff
@@ -57,9 +57,18 @@ declare namespace electronPackager {
     }
 
     interface ElectronOsXSignOptions {
-        identity?: string;
         entitlements?: string;
         "entitlements-inherit"?: string;
+        identity?: string | boolean;
```

Comment:
[electron-osx-sign](https://www.npmjs.com/package/electron-osx-sign#opts) docs say that identity is till a string. Could you clarify?

Reply:
Please see `mac.js` lines 356-361:

https://github.com/electron-userland/electron-packager/blob/92d09bba34599283a794fd6f24b88470f0cb1074/src/mac.js#L356-L361

Reply:
awesome, thanks!

Reply:
The more I think about it, it should actually be `string | true`, what do you think?

Reply:
Definitely, well spotted. There doesn't seem to be logic for handling `false`. Could you also add an inline comment with the link you shared here for future reference?

Reply:
Done :slightly_smiling_face:

Reply:
:+1: